src/org/thoughtcrime/securesms/WebViewActivity.java

```diff
@@ -122,6 +127,22 @@ public WebResourceResponse shouldInterceptRequest(WebView view, WebResourceReque
         return super.shouldInterceptRequest(view, request);
       }
     });
+    webView.setWebChromeClient(new WebChromeClient() {
+      @Override
+      @RequiresApi(21)
+      public boolean onShowFileChooser(WebView webView, ValueCallback<Uri[]> filePathCallback, WebChromeClient.FileChooserParams fileChooserParams) {
+        if (mFileUploadCallbackSecond != null) {
+          mFileUploadCallbackSecond.onReceiveValue(null);
+        }
+        mFileUploadCallbackSecond = filePathCallback;
+        Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
+        intent.addCategory(Intent.CATEGORY_OPENABLE);
+        intent.setType("*/*");
```

Comment:
Is there something in fileChooserParams for this too?

Reply:
after checking: there is `WebChromeClient.FileChooserParams.getAcceptTypes()`, not sure if that is 100% compatible to java, however - i would expect AdvancedWebView to use that as well otherwise - so, maybe play safe for now and focus on mimicing AdvancedWebView and improve in a subsequent pr at some later point

Reply:
EXTRA_MIME_TYPES comes from an intent, but we do not get an intent from onShowFileChooser(), so that would not help anyways as far as i understand it

Reply:
ah, yes, i understand. so passing `getAcceptTypes()` to EXTRA_MIME_TYPES and/or `setType()` - but yes, this is sth. for another iteration i'd say, again, there might be reasons, `AdvancedWebView` did not do it if it is apparently simple :)